stable/kuberhealthy/Chart.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+appVersion: "0.1.1"
+home: https://comcast.github.io/kuberhealthy/
+description: The official Helm chart for Kuberhealthy.
+name: kuberhealthy
+version: 0.1.1
+maintainers:
+  - name: integrii
+    email: eric.greer@comcast.com
+  - name: lolimjake
+    email: jacob.martin@comcast.com
+keywords:
+- kuberhealthy
+- kubernetes
+- monitoring
+- synthetic
+- comcast
+- health
+- cluster
+sources:
+- https://github.com/Comcast/kuberhealthy
```

Comment:
There can be multiple `sources`. Including the chart is useful, particularly in <https://hub.helm.sh>

Reply:
Added.